### PR TITLE
Update /desktop/developers for 22.10 release

### PR DESCRIPTION
**Pull request by carkod**
## Done

Update desktop/developers

## QA

**Only 22.10 updates**

- Go to https://ubuntu-com-12166.demos.haus/desktop/developers and compare with [copy doc](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit#heading=h.b332hqeenxjz)
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://github.com/canonical/web-squad/issues/6083
Fixes https://github.com/canonical/web-squad/issues/6109

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -501,6 +501,32 @@ <h3>Certified hardware</h3>
       </div>
     </div>
   </section>
+  <section class="p-strip--light">
+    <div class="row">
+      <div class="col-8">
+        <h2>What's new in Ubuntu {{ releases.lts.short_version }}</h2>
```
Comment (minkyngkm) [inline templates/desktop/developers.html:507]: This needs to say 22.10 but it says 22.04 now... is it gonna be updated automatically?

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -501,6 +501,32 @@ <h3>Certified hardware</h3>
...
+        <li class="p-list__item is-ticked">Pipewire as the default audio system delivering improve bluetooth performance</li>
+        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.1, Perl 5.34, Go 1.18, GCC 12.2, Rust 1.59</li>
+        <li class="p-list__item is-ticked">Core productivity apps LibreOffice 7.4, Thunderbird 102, and Firefox 104</li>
+        <li class="p-list__item is-ticked">Multi-threaded CPU squashfs decompression for faster Snap startup</li>
+      </ul>
+    </div>
+    <div class="u-fixed-width">
+      <p><a href="{{ releases.lts.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
+    </div>
+  </section>
```
Comment (minkyngkm) [inline templates/desktop/developers.html:529]: Do you think we need to add a line between 22.10 section and 22.04 section? 
```suggestion
  </section>
   <div class="u-fixed-width">
      <hr class="p-separator" />
    </div>
```

File: templates/desktop/developers.html (hunk 1 above)
Comment (carkod) [reply templates/desktop/developers.html:507]: yeah, this is changed in releases.yaml in [this PR](https://github.com/canonical/ubuntu.com/pull/12174/files) that's why this PR is mergeing into a feature branch `[22-10-release](https://github.com/canonical/ubuntu.com/tree/22-10-release)`

File: templates/desktop/developers.html (hunk 2 above)
Comment (carkod) [reply templates/desktop/developers.html:529]: Added

Comment (carkod) [reply templates/desktop/developers.html:529]: well, I'd rather not if they are going to take 2 days to review a line...

File: templates/desktop/developers.html (hunk 1 above)
Comment (carkod) [reply templates/desktop/developers.html]: I moved the yaml file to this PR, so please can you review

**Comment by carkod** [comment]
@minkyngkm I can't get the demo to work, so you'll have to test locally

File: templates/desktop/developers.html [hunk 3]
```diff
@@ -502,6 +502,33 @@ <h3>Certified hardware</h3>
...
+            <li>New quick settings interface</li>
+            <li>New spread effect for multitasking</li>
+            <li>Dynamic UI for the file manager</li>
+          </ul>
+        </li>
+        <li class="p-list__item is-ticked">Pipewire as the default audio system delivering improve bluetooth performance</li>
+        <li class="p-list__item is-ticked">Up-to-date toolchains, including Python 3.10, PHP 8.1, Ruby 3.1, Perl 5.34, Go 1.19, GCC 12.2, Rust 1.61</li>
+        <li class="p-list__item is-ticked">Core productivity apps LibreOffice 7.4, Thunderbird 102, and Firefox 105</li>
+        <li class="p-list__item is-ticked">Multi-threaded CPU squashfs decompression for faster Snap startup</li>
+        <li class="p-list__item is-ticked">Preview access to Azure Active Directory user authentication</li>
```
Comment (minkyngkm) [inline templates/desktop/developers.html:524]: Waiting on Oliver's reply in the copy doc.

File: templates/desktop/developers.html [hunk 4]
```diff
@@ -502,6 +502,33 @@ <h3>Certified hardware</h3>
...
+      </ul>
+    </div>
+    <div class="u-fixed-width">
+      <p><a href="{{ releases.latest.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
```
Comment (minkyngkm) [inline templates/desktop/developers.html:528]: Are we keep this release note as its not found in the copy doc.

File: templates/desktop/developers.html [hunk 5]
```diff
@@ -539,9 +566,12 @@ <h2>What's new in Ubuntu {{ releases.lts.short_version }}</h2>
       </ul>
     </div>
     <div class="u-fixed-width">
-      <p><a href="{{ releases.lts.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
+      <p><a class="p-link--inverted" href="{{ releases.lts.release_notes_url }}">Read the full release notes&nbsp;&rsaquo;</a></p>
     </div>
   </section>
+  <div class="u-fixed-width">
+    <hr class="p-separator" />
+  </div>
```
Comment (minkyngkm) [inline templates/desktop/developers.html:574]: This line needs to be deleted.
```suggestion
 
```

**Review by minkyngkm — APPROVED** [review]
LGTM @carkod ! 👍 I've just made some comments for tiny things. Also, I'm wondering if we need to make an issue to update the graph in the 'Why choose Ubuntu Desktop?" section.

**Comment by carkod** [comment]
> LGTM @carkod ! +1 I've just made some comments for tiny things. Also, I'm wondering if we need to make an issue to update the graph in the 'Why choose Ubuntu Desktop?" section.

Good point. It looks like it's updated across the site in the charts-data.js? Not 100% yet but I'll have to look into in another PR

**Pull request merged**